app/api/tests/test_api.py

```diff
@@ -1199,6 +1201,147 @@ def test_can_filter_doc_without_anotation(self):
+    def setUpTestData(cls):
+        cls.super_user_name = 'super_user_name'
+        cls.super_user_pass = 'super_user_pass'
+        create_default_roles()
+        super_user = User.objects.create_superuser(username=cls.super_user_name,
+                                                   password=cls.super_user_pass,
+                                                   email='fizz@buzz.com')
+        cls.classification_project = mommy.make('TextClassificationProject',
+                                                users=[super_user], project_type=DOCUMENT_CLASSIFICATION)
+
+        cls.classification_upload_url = reverse(viewname='doc_uploader', args=[cls.classification_project.id])
+        cls.classification_download_url = reverse(viewname='doc_downloader', args=[cls.classification_project.id])
+        assign_user_to_role(project_member=super_user, project=cls.classification_project,
+                            role_name=settings.ROLE_PROJECT_ADMIN)
+
+        # Add other projects
+        cls.labeling_project = mommy.make('SequenceLabelingProject',
+                                          users=[super_user], project_type=SEQUENCE_LABELING)
+        cls.labeling_upload_url = reverse(viewname='doc_uploader', args=[cls.labeling_project.id])
+        cls.labeling_download_url = reverse(viewname='doc_downloader', args=[cls.labeling_project.id])
+        assign_user_to_role(project_member=super_user, project=cls.labeling_project,
+                            role_name=settings.ROLE_PROJECT_ADMIN)
+
+        cls.seq2seq_project = mommy.make('Seq2seqProject', users=[super_user], project_type=SEQ2SEQ)
+        cls.seq2seq_upload_url = reverse(viewname='doc_uploader', args=[cls.seq2seq_project.id])
+        cls.seq2seq_download_url = reverse(viewname='doc_downloader', args=[cls.seq2seq_project.id])
+        assign_user_to_role(project_member=super_user, project=cls.seq2seq_project,
+                            role_name=settings.ROLE_PROJECT_ADMIN)
+
+    def setUp(self):
+        self.client.login(username=self.super_user_name,
+                          password=self.super_user_pass)
+
+    def load_test_helper(self, upload_url, download_url, filename, import_format, project, export_format=None, **kwargs):
+        if not export_format:
+            export_format = import_format
+
+        import_data = []
+        parser = TextUploadAPI.select_parser(import_format)
+
+        with open(os.path.join(DATA_DIR, filename), 'rb') as f:
+            self.client.post(upload_url, data={'file': f, 'format': import_format})
+            f.seek(0)
+            imported = parser.parse(f)
+            import_data = [x for elem in imported for x in elem]
+
+        label_mapping = {label.id: label.text for label in project.labels.all()}
+        import_data = [x for x in import_data if x.get('labels')]
+
+        if export_format == 'json':
+            r = self.client.get(download_url, data={'q': export_format}, HTTP_ACCEPT="application/json")
```

Comment:
please remove the `if` here and add another parameter to the method for the HTTP_ACCEPT header